payroll/views.py

```diff
@@ -103,12 +119,34 @@ def get_context_data(self, **kwargs):
         return context
 
     def median_entity_salary(self):
-        q = Salary.objects.filter(Q(job__position__employer__parent=self.object) | Q(job__position__employer=self.object))
+        q = Salary.objects.filter(
+            Q(job__position__employer__parent=self.object) |
+            Q(job__position__employer=self.object)
+        )
 
         results = q.all().aggregate(median=Percentile('amount', 0.5, output_field=FloatField()))
 
         return results['median']
 
+    def _make_pie_chart(self, container):
+        base_pay, extra_pay = self.get_entity_payroll()
```

Comment:
Getting these values seems separate from making the pie chart object. How do you feel about passing base pay and extra pay as arguments to this function?